server/catalog/table.cpp

```diff
@@ -23,6 +23,7 @@
 
 #include <absl/algorithm/container.h>
 #include <absl/strings/str_cat.h>
+#include <velox/type/Type.h>
```

Comment:
why do you add random includes? it's not first time when I noticed this.
You probably need to specify this argument for clangd:
```
--header-insertion=never
```